create aleph alpha integration doc

davidmyriel opened the pull request:
This integration doc is built on top of our basic integration and shows how to do an image search based on a common dataset. I wonder if we should have our own demo, though.

I still need to fix some wording and make it a subsection of aleph alpha.

llogiq commented:
I think Aleph Alpha is going to do their own demo, so perhaps we don't need to do the work twice.

davidmyriel commented:
> Let's keep pages consistent? ![image](https://user-images.githubusercontent.com/1935623/246363792-ddc18c12-ee02-467e-81df-9b39264fd44d.png)

I have already fixed this yesterday.

llogiq commented on qdrant-landing/content/documentation/integrations/aleph-alpha-search.md:
```diff
@@ -0,0 +1,185 @@
+1. A Qdrant instance has to be running. If you want to launch it locally,
+   [Docker is the fastest way to do that](https://qdrant.tech/documentation/quick_start/#installation).
+2. You need to have a [Aleph Alpha account registered and confirmed](https://app.aleph-alpha.com/). Once it’s done, 
+   the API key has to be created (see: [API Tokens](https://app.aleph-alpha.com/profile)).
```
Perhaps use active voice, i.e. "Once that's done, create an API key (...)" here.

llogiq commented on qdrant-landing/content/documentation/integrations/aleph-alpha-search.md:
```diff
@@ -0,0 +1,185 @@
+2. You need to have a [Aleph Alpha account registered and confirmed](https://app.aleph-alpha.com/). Once it’s done, 
+   the API key has to be created (see: [API Tokens](https://app.aleph-alpha.com/profile)).
+
+Once it’s done we can store the Aleph Alpha API key in a variable and choose the model we’re going to use.
```
```suggestion
Now we can store the Aleph Alpha API key in a variable and choose the model we’re going to use.
```

llogiq reviewed (commented):
I've added a few small suggestions. Apart from that, do we go for the "we" form or just let "you", the reader do stuff? I have done both in the past, perhaps @davidmyriel has a clearer vision here.

davidmyriel commented:
Shortened text in tutorial format and moved to Tutorials. 
Turned to active voice and changed titles.
I decided to keep integrations strictly for basic reference docs. 
More structure for tutorials/integrations covered in a PR later today.
